src/e2e-test/java/io/cdap/plugin/bigquery/actions/BigQueryActions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.bigquery.actions;
+
+import io.cdap.e2e.pages.locators.CdfBigQueryPropertiesLocators;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.SeleniumHelper;
+
+/**
+ * BigQuery plugin step actions.
+ */
+public class BigQueryActions {
```

Comment:
remove this class, similar class is present in `cdap-e2e-tests` repo, any changes should be maintained at one place.

Reply:
Removed.